lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -23,9 +23,11 @@ use OpenQA::Schema::Result::JobDependencies;
 use OpenQA::Utils qw(determine_web_ui_web_socket_url get_ws_status_only_url);
 use Mojo::ByteStream;
 use Mojo::Util 'xml_escape';
+use Mojo::File 'path';
 use File::Basename;
 use POSIX 'strftime';
 use Mojo::JSON 'to_json';
+use Mojo::JSON 'decode_json';
```

Comment:
Please combine includes from the same file within one line:

```
use Mojo::JSON qw(to_json decode_json);
```

Reply:
Done